.github/workflows/gradle-pr-build.yml

```diff
@@ -3,6 +3,11 @@ name: PR Build
 on:
   pull_request:
 
+env:
+  ORG_GRADLE_PROJECT_signingKey : ${{ secrets.ORG_GRADLE_PROJECT_SIGNINGKEY }}
+  ORG_GRADLE_PROJECT_signingKeyId: ${{ secrets.ORG_GRADLE_PROJECT_SIGNINGKEYID }}
+  ORG_GRADLE_PROJECT_signingKeyPassword : ${{ secrets.ORG_GRADLE_PROJECT_SIGNINGPASSWORD }}
```

Comment:
@shashachu I believe these secrets are unnecessary for PR builds for now, right?

Reply:
If so, I'll push a commit the remove them.

Reply:
no, unfortunately, at least without some other tweaking. These being missing were the reason why the PR builds started failing once @paul-dingemans committed the version to be a non-SNAPSHOT version.

Reply:
Gotcha.